notebooks/summit/AuxTel/SITCOM-627/SITCOM-627_Measure_Astigmatism.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<p>Why does it say that CWFS will not work? </p><p>Did you run this on the TTS without setting up the environment?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I think we'd have to look together, but if you run the summit environment setup script at the TTS, and you make sure your <code>~/notebooks/.user_setups</code> file doesn't have any packages being setup, then you shouldn't see this. </p>

Reply:
Well, it was working well in the past weeks and I didn't change anything in the environment. Yesterday, I decided to rerun my summit environment, to start from scratch, but the TTS doesn't have the required nublado image so the bash script quits.